x/gov/keeper/tally_test.go

```diff
@@ -242,10 +242,10 @@ func TestTallyDelegatorInherit(t *testing.T) {
 
 	coin, err := sdk.ParseDecCoin("11000" + common.NativeToken)
 	require.Nil(t, err)
-	delegator1Msg := staking.NewMsgDelegate(Addrs[3], coin)
+	delegator1Msg := staking.NewMsgDeposit(Addrs[3], coin)
 	stakingHandler(ctx, delegator1Msg)
 
-	voteMsg := staking.NewMsgVote(Addrs[3], []sdk.ValAddress{sdk.ValAddress(Addrs[2])})
+	voteMsg := staking.NewMsgAddShares(Addrs[3], []sdk.ValAddress{sdk.ValAddress(Addrs[2])})
```

Comment:
voteMsg := staking.NewMsgAddShares

the returned variable should be changed to 'addSharesMsg'

Reply:
appreciated!